src/main/docker/kafka/supervisor/kafka.conf

```diff
@@ -0,0 +1,4 @@
+[program:kafka]
```

Comment:
If this is copied from somewhere please provide a reference at the top and include their copyright header. If you changed this substantially, please include our copyright header and a post-copyright comment regarding the original source.